sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1800,6 +1800,24 @@ object functions {
    */
   def cosh(columnName: String): Column = cosh(Column(columnName))
 
+  /**
+   * @param e angle in radians
+   * @return cotangent of the angle
+   *
+   * @group math_funcs
+   * @since 3.3.0
+   */
+  def cot(e: Column): Column = withExpr { Cot(e.expr) }
+
+  /**
+   * @param columnName angle in radians
+   * @return cotangent of the angle
+   *
+   * @group math_funcs
+   * @since 3.2.0
+   */
+  def cot(columnName: String): Column = cot(Column(columnName))
```

Comment:
Mind removing this one? Looks fine otherwise

Reply:
Thanks for the reminder. I just removed it.